### PR TITLE
Take screenshots when ui tests failed

### PR DESCRIPTION
**Pull request by IlyaEremin**

### CODE REVIEW

**Comment by IlyaEremin** [comment]
@RamilGabdrakhmanov @almaziskhakov @yaroslavsudnik review required

File: app/build.gradle [hunk 1]
```diff
@@ -76,6 +78,10 @@ android {
         lintConfig file("$projectDir/lint.xml")
         warningsAsErrors true
     }
+
+    configurations.all {
```
Comment (RamilGabdrakhmanov) [inline app/build.gradle]: its necessary?

File: circle.yml [hunk 2]
```diff
@@ -1,19 +1,44 @@
...
 dependencies:
   pre:
+    - chmod +x app/set_animation_permissions.sh
     - echo y | android update sdk --no-ui --all --filter "tools,platform-tools,android-25,extra-google-m2repository"
     - echo y | android update sdk --no-ui --all --filter "build-tools-25.0.1"
     - echo y | android update sdk --no-ui --all --filter "extra-android-m2repository"
 
+  post:
+    # Create the android emulator
+    - echo n | android create avd -n test_android_17 -f -t android-17
```
Comment (RamilGabdrakhmanov) [inline circle.yml]: Android 17? Could we use circleci-android22?

Comment (IlyaEremin) [inline circle.yml]: android 17 should be way faster than build in circleci-android22.
There is no difference between 17 and 22 in term of behaviour. 
New permission system was introduced in 23.

File: app/build.gradle (hunk 1 above)
Comment (IlyaEremin) [inline app/build.gradle]: @RamilGabdrakhmanov no. This is example of how to resolve conflict between test and main dependencies

**Pull request merged**